code/game/objects/items/weapons/shields.dm

```diff
@@ -41,12 +45,7 @@
 	icon_state = "buckler"
 	item_state = "buckler"
 	materials = list()
-
-/obj/item/weapon/shield/riot/buckler/IsShield()
-	if(prob(60))
-		return 1
-	else
-		return 0
+	block_chance = 60

```

Comment:
I believe this is meant to have a 30% chance, not 60%-it's worse than a riot shield, not better.